src/api/helpers/send-email-application-submission-deadline/send-email/index.ts

```diff
@@ -0,0 +1,33 @@
+import mapApplicationSubmissionDeadlineVariables from '../../map-application-submission-deadline-variables';
+import sendEmail from '../../../emails';
+import { Application } from '../../../types';
+
+/**
+ * send
+ * generates an array of promises to send submission deadline emails
+ * resolves all promises and logs the number of emails sent
+ * @param {Array<Application>} applications: Array of applications
+ * @returns {Promise<void>} application submission deadline emails sent
+ */
+const send = async (applications: Array<Application>): Promise<void> => {
```

Comment:
What's the reason for `Promise<void>`?

Reply:
was the only way lint did not complain

Reply:
What was the lint issue?

Reply:
Type 'void' is not assignable to type 'void[]' or whatever you try to cast it to